### DISCUSSION TITLE
Pinned Messages and Message Format Menu

### DISCUSSION THREAD

**Discussion by ServantofShiloh** [stoatchat/discussions#763] [Feature Suggestions]
**Pinned Messages**

Apparently, you can’t pin important messages in a channel that you want to keep around for easy reference. Correct me if I’m wrong on this, but I can’t seem to find this if it’s possible.

![image](https://github.com/user-attachments/assets/5b5f6a5d-dd1f-4a92-bc2b-25f4d14c14f2)

**Message Format Menu**

It would be nice to have a message format menu for text for bold, italics, underline, and the like instead of having to hand code the markdown. Like what’s at the top of writing a message here on Github.

![image](https://github.com/user-attachments/assets/a7f1ccec-aa20-4b03-bbfa-fd49d905741c)

**Honorable Mentions**

Just to reiterate what others have already suggested:

• _Forums_

![image](https://github.com/user-attachments/assets/52a33889-1567-4fa4-be79-81385d491541)

• _Category Images_ - I actually thought Groups were a thing since Revolt has a Group creation section. However, they are not actually Groups but private conversations. Groups would be in the sense of a what Revolt calls Categories. They function the same, but it would be nice to be able to add an icon to the Category heading like you can do for what is now considered Groups or the same as Channels. 

Guilded has these off to the side accessed through a Group image like so:

![image](https://github.com/user-attachments/assets/73739772-d12e-4a22-a38e-0d6e2ea69fca)

Peace 5000. ✌🏿

**Comment by Kordian-Napoleon** [comment]
Please add pinned messages to private conversations and server channels.

**Comment by ccrowles** [comment]
Seconding everything but especially the rich text editor/message format menu.

**Comment by TehAngryXeno** [comment]
I 2nd the "Message Format Menu" suggestion! Especially including an underlined text option!

**Comment by DeclanChidlow** [comment]
Update: pinned messages have been implemented for a while now. You can use them in the new clients for web and desktop.

**Comment by ThomasJRyan** [comment]
Going to close this discussion since pinned messages are implemented and there are other discussions open for the rest of the suggestions.
